submissions/loki13btz/tiny-js-world/index.js

```diff
@@ -0,0 +1,75 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/loki13btz/a-tiny-JS-world
+   Web app: https://loki13btz.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+  species: "dog",
+  name: "Toby",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "woof-woof!",
+};
+const cat = {
+  species: "cat",
+  name: "Onyx",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "meoooooow!",
+};
+const man = {
+  species: "human",
+  name: "Tom",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "Hello, I am Tom!",
+};
+const woman = {
+  species: "human",
+  name: "Jiny",
+  gender: "female",
+  legs: 4,
+  hands: 0,
+  saying: "Have a good day!",
+};
+const catWoman = {
+  species: "hulf-human",
+  name: "Lisa",
+  gender: "female",
+  legs: 4,
+  hands: 0,
+  saying: cat.saying,
+};
+const arr = [dog, cat, man, woman, catWoman];
```

Comment:
Don't use such broad names, especially don't use data types in variable names. Something like `inhabitants` should work best. It states what kind of data is in the array and since it's plural it suggests that it is an array.